CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+For technical issues related to using Node.js that are not issues with the
```

Comment:
Committed. Thanks for being awesome!

On Tue, Oct 6, 2015 at 11:07 PM Rich Trott <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/nodejs/node/pull/3233#discussion_r41354508>:
>
> > @@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
> >  please make sure discussions are related to concrete technical issues with the
> >  Node.js software.
> >
> > +For technical issues related to using Node.js that are not issues with the
>
> Yes, go ahead with a second commit on top of this one. (You or I or
> someone else can squash 'em all down into one commit later.)
>
> Thanks for tackling this, by the way!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/3233/files#r41354508>.
>
-- 
Best Regards,

Doug Shamoo